### PR TITLE
Brace style fixes

### PR DESCRIPTION
**Pull request by Amelia55555**
Applies brace style fixes, thanks to my new rules I'm writing for Stylecop.

### CODE REVIEW

**Comment by GeertJohan** [comment]
What is the rationale behind the removing of `{` and `}` tokens at if/else/for statements with a single following statement?

Code like this becomes hard to read: https://github.com/ScottNZ/OpenRA/blob/stylecop-fixes/OpenRA.Mods.Common/UtilityCommands/UpgradeRules.cs#L225-LL227

And it's also a cause of bugs: http://embeddedgurus.com/barr-code/2014/03/apples-gotofail-ssl-security-bug-was-easily-preventable/

I'm not a C# coder, so if this is just the accepted C# style then never mind. I don't want to start a huge styling debate here..

**Comment by ScottNZ** [comment]
It helps bring this code in line with the rest of the code, style convention-wise.

**Comment by pchote** [comment]
It's part of our [coding standard](https://github.com/OpenRA/OpenRA/wiki/Coding-Standard), and makes for concise code.  Many of the arguments for enforcing braces around single lines come down to "it makes sloppy code safer", but we aim to avoid sloppy code in the first place :)

File: OpenRA.Mods.RA/Widgets/ObserverProductionIconsWidget.cs [hunk 1]
```diff
@@ -60,9 +58,7 @@ public override void Draw()
 			foreach (var queue in queues)
 			{
 				if (!clocks.ContainsKey(queue.Trait))
-				{
 					clocks.Add(queue.Trait, new Animation(world, "clock"));
-				}
 			}
```
Comment (penev92) [inline OpenRA.Mods.RA/Widgets/ObserverProductionIconsWidget.cs]: You missed these.

Comment (ScottNZ) [inline OpenRA.Mods.RA/Widgets/ObserverProductionIconsWidget.cs]: Nope, intentional

Comment (ScottNZ) [inline OpenRA.Mods.RA/Widgets/ObserverProductionIconsWidget.cs]: Actually, I'll change this

**Comment by penev92** [comment]
> I'm not a C# coder, so if this is just the accepted C# style then never mind.

No, not really.

I'm not a fan of the braces cull, but since it is inevitable, I might as well give a :+1:

**Comment by GeertJohan** [comment]
@ScottNZ @pchote 
I'm aware of the convention/coding standard in this project. I was just wondering WHY it was the standard... 
It's good that there is a strong styling convention. I come from a language that has very strict styling rules (Go). So yeah, all for it!

**Comment by GeertJohan** [comment]
@penev92 Yeah I also prefer the Go styling:
```
if a ==  42 {
 But it's better to follow one

**Comment by Phrohdoh** [comment]
:+1:

**Comment by Phrohdoh** [comment]
Thanks!

**Pull request merged**